[2021-12-08 16:45:38] request.INFO: Matched route "Liste_Dossiers_Controller/visualiserDossier". {"route":"Liste_Dossiers_Controller/visualiserDossier","route_parameters":{"_route":"Liste_Dossiers_Controller/visualiserDossier","_controller":"App\\Controller\\FormationDossierController::show","id":"29189"},"request_uri":"http://192.168.0.29/dossier/29189/visualiser","method":"GET"} []
[2021-12-08 16:45:38] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-08 16:45:38] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-08 16:45:38] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascal"} []
[2021-12-08 16:45:38] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-08 16:45:38] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:38] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:38] php.CRITICAL: Call to a member function getDateEnvoi() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:38] request.CRITICAL: Uncaught PHP Exception Error: "Call to a member function getDateEnvoi() on null" at /var/www/src/Controller/FormationDossierController.php line 622 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:38] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-08 16:45:47] request.INFO: Matched route "Liste_Dossiers_Controller/visualiserDossier". {"route":"Liste_Dossiers_Controller/visualiserDossier","route_parameters":{"_route":"Liste_Dossiers_Controller/visualiserDossier","_controller":"App\\Controller\\FormationDossierController::show","id":"29189"},"request_uri":"http://192.168.0.29/dossier/29189/visualiser","method":"GET"} []
[2021-12-08 16:45:47] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-08 16:45:47] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-08 16:45:47] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascal"} []
[2021-12-08 16:45:47] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-08 16:45:47] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:47] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:47] php.CRITICAL: Call to a member function getDateEnvoi() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:47] request.CRITICAL: Uncaught PHP Exception Error: "Call to a member function getDateEnvoi() on null" at /var/www/src/Controller/FormationDossierController.php line 622 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:47] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-08 16:45:52] request.INFO: Matched route "Liste_Dossiers_Controller/visualiserDossier". {"route":"Liste_Dossiers_Controller/visualiserDossier","route_parameters":{"_route":"Liste_Dossiers_Controller/visualiserDossier","_controller":"App\\Controller\\FormationDossierController::show","id":"29189"},"request_uri":"http://192.168.0.29/dossier/29189/visualiser","method":"GET"} []
[2021-12-08 16:45:52] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-08 16:45:52] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-08 16:45:52] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascal"} []
[2021-12-08 16:45:52] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-08 16:45:52] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:52] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-08 16:45:52] php.CRITICAL: Call to a member function getDateEnvoi() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:52] request.CRITICAL: Uncaught PHP Exception Error: "Call to a member function getDateEnvoi() on null" at /var/www/src/Controller/FormationDossierController.php line 622 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getDateEnvoi() on null at /var/www/src/Controller/FormationDossierController.php:622)"} []
[2021-12-08 16:45:52] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-13 10:41:04] request.INFO: Matched route "home.portail". {"route":"home.portail","route_parameters":{"_route":"home.portail","_controller":"App\\Controller\\HomeController::portail"},"request_uri":"http://192.168.0.29/portail","method":"GET"} []
[2021-12-13 10:41:04] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-13 10:41:04] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-13 10:41:04] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascalf"} []
[2021-12-13 10:41:04] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-13 10:41:04] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:41:04] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:41:04] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "note.update" as such route does not exist.")." at /var/www/templates/home/portail.html.twig line 247 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"note.update\" as such route does not exist.\"). at /var/www/templates/home/portail.html.twig:247, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"note.update\" as such route does not exist. at /var/www/vendor/symfony/routing/Generator/CompiledUrlGenerator.php:50)"} []
[2021-12-13 10:41:04] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-13 10:41:33] request.INFO: Matched route "home.portail". {"route":"home.portail","route_parameters":{"_route":"home.portail","_controller":"App\\Controller\\HomeController::portail"},"request_uri":"http://localhost/portail","method":"GET"} []
[2021-12-13 10:41:33] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-13 10:41:33] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-13 10:41:33] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascal"} []
[2021-12-13 10:41:33] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-13 10:41:33] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:41:33] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:41:33] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "note.update" as such route does not exist.")." at /var/www/templates/home/portail.html.twig line 247 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"note.update\" as such route does not exist.\"). at /var/www/templates/home/portail.html.twig:247, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"note.update\" as such route does not exist. at /var/www/vendor/symfony/routing/Generator/CompiledUrlGenerator.php:50)"} []
[2021-12-13 10:41:33] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-13 10:47:05] request.INFO: Matched route "home.portail". {"route":"home.portail","route_parameters":{"_route":"home.portail","_controller":"App\\Controller\\HomeController::portail"},"request_uri":"http://localhost/portail","method":"GET"} []
[2021-12-13 10:47:05] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2021-12-13 10:47:05] php.INFO: User Deprecated: Creating Doctrine\ORM\Mapping\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Creating Doctrine\\ORM\\Mapping\\UnderscoreNamingStrategy without making it number aware is deprecated and will be removed in Doctrine ORM 3.0. at /var/www/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/UnderscoreNamingStrategy.php:66)"} []
[2021-12-13 10:47:05] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"Pascal"} []
[2021-12-13 10:47:05] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-12-13 10:47:05] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:47:05] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\CrmAprentivAuthenticator"} []
[2021-12-13 10:47:05] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "note.update" as such route does not exist.")." at /var/www/templates/home/portail.html.twig line 247 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"note.update\" as such route does not exist.\"). at /var/www/templates/home/portail.html.twig:247, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"note.update\" as such route does not exist. at /var/www/vendor/symfony/routing/Generator/CompiledUrlGenerator.php:50)"} []
[2021-12-13 10:47:05] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2021-12-13 10:47:33] console.ERROR: Error thrown while running command "cache:clear -e prod". Message: "Unable to write in the "/home/jeremy/Bureau/project/app/var/cache/prod" directory" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): Unable to write in the \"/home/jeremy/Bureau/project/app/var/cache/prod\" directory at /home/jeremy/Bureau/project/app/vendor/symfony/framework-bundle/Command/CacheClearCommand.php:88)","command":"cache:clear -e prod","message":"Unable to write in the \"/home/jeremy/Bureau/project/app/var/cache/prod\" directory"} []
[2021-12-13 10:47:33] console.DEBUG: Command "cache:clear -e prod" exited with code "1" {"command":"cache:clear -e prod","code":1} []
